mocks/routes/preset.js

```diff
@@ -0,0 +1,153 @@
+let presets = [
+        {
+        id: 1,
+        name: "default preset",
+        thresholds: [
+            {
+                type: "temperature",
+                min: 27.5,
+                max: 32.2
+            },
+            {
+                type: "humidity",
+                min: 60.5,
+                max: 87.5
+            },
+            {
+                type: "co2",
+                min: 12,
+                max: 20
+            },
+        ]
+    },
+        {
+        id: 2,
+        name: "fallback preset",
+        thresholds: [
+            {
+                type: "temperature",
+                min: 25.0,
+                max: 35.0
+            },
+            {
+                type: "humidity",
+                min: 30.0,
+                max: 90.0
+            },
+            {
+                type: "co2",
+                min: 10,
+                max: 30
+            },
+        ]
+    }
+]
+
+module.exports = [
+    {
+    id: "get-presets", // id of the route
+    url: "/api/preset", // url in path-to-regexp format
+    method: "GET", // HTTP method
+    variants: [
+      {
+        id: "success", // id of the variant
+        type: "json", // variant type
+        options: {
+          status: 200,
+          body: presets
+        }
+      },
+    ]
+    },
+    {
+        id: "add-presets", // route id
+        url: "/api/preset", // url in express format
+        method: "POST", // HTTP method
+        variants: [
+          {
+            id: "success",
+            type: "middleware", // variant of type "middleware"
+            options: {
+              middleware: (req, res) => {
+                    presets.push(req.body.preset)
```

Comment:
Here the body object is the preset object and does not have a preset object in it.
```suggestion
                    presets.push(req.body)
```